readme.md

```diff
@@ -58,7 +57,7 @@ Note that instructors will *not* be grading on a numbered scale this time. Proje
 
 Attendance will be at 12:00pm on Friday. Groups are otherwise welcome to work outside of GA at their collective discretion.
 
-You group **must** designate a "representative" to contact an instructor 
+You group **must** designate a "representative" to contact an instructor.  ???mms To do what?  When?
```

Comment:
I vote remove. My understanding was we wouldn't mention the Typeform to the students until Monday, or something?